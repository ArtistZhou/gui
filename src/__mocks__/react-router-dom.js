import React from 'react';
import { compose, setDisplayName } from 'recompose';
import * as rrd from 'react-router-dom';
// Just render plain div with its children
rrd.BrowserRouter = compose(setDisplayName('BrowserRouter'))(({ children }) => <div>{children}</div>);

module.exports = rrd;
