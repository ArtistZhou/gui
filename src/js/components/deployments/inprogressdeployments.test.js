import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Progress from './inprogressdeployments';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <Progress progress={[]} />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
