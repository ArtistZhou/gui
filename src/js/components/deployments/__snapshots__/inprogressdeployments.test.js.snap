// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InProgressDeployments Component renders correctly 1`] = `"<div></div>"`;
